graph_spec.rb

```diff
@@ -52,5 +52,24 @@
 
       expect(g.topological_sort).to eq(expected)
     end
+
+    it 'raises an UnsortableCyclicGraphError when the graph has cycles' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.